test/Nancy.Tests.Functional/project.json

```diff
@@ -0,0 +1,25 @@
+{
+    "dependencies": {
+        "Nancy": { "target": "project" },
+        "Nancy.Testing": { "target": "project" },
+        "Nancy.ViewEngines.Razor": { "target": "project" },
+        "xunit": "2.1.0",
+        "xunit.runner.dnx": "2.1.0-*"
+    },
+
+    "commands": {
+        "test": "xunit.runner.dnx"
+    },
+
+    "compilationOptions": {
+        "define": [ "DNX" ]
+    },
+
+    "compileFiles": [
+        "../Nancy.Tests/xUnitExtensions/RecordAsync.cs"

```

Comment:
OK, from the documentation it seems it'll glob `compile` and `compileBuiltIn` (both allows wildcards), exclude `compileExclude` and then add the individual `compileFiles`. So there should be no difference, other than `compile` allows globbing, and `compileFiles` doesn't.